content/hacking-atom/sections/package-word-count.md

```diff
@@ -51,7 +51,11 @@ style sheets your package needs to load. If not specified, style sheets in the `
 * `menus`: an Array of Strings identifying the order of the menu mappings your package needs to load. If not specified, mappings in the `menus` directory are added alphabetically.
 * `snippets`: an Array of Strings identifying the order of the snippets your package needs to load. If not specified, snippets in the `snippets` directory are added alphabetically.
 * `activationCommands`: an Object identifying commands that trigger your package's activation. The keys are CSS selectors, the values are Arrays of Strings identifying the command. The loading of your package is delayed until one of these events is triggered within the associated scope defined by the CSS selector.  If not specified, the `activate()` method of your main export will be called when your package is loaded.
-* `activationHooks`: an Array of Strings identifying hooks that trigger your package's activation. The loading of your package is delayed until one of these hooks are triggered. Currently, there are two activation hooks: `language-package-name:grammar-used` (e.g., `language-javascript:grammar-used`) and `core:loaded-shell-environment`.
+* `activationHooks`: an Array of Strings identifying hooks that trigger your package's activation. The loading of your package is delayed until one of these hooks are triggered. Currently, there are three activation hooks:
+  *  `language-package-name:grammar-used` (e.g., `language-javascript:grammar-used`),
+  * `language-scope:root-scope-used` (e.g., `javascript:root-scope-used` or `source.gfm:root-scope-used`), and
```

Comment:
`source.javascript`, no?

Reply:
That was supposed to be a Tree-sitter example. I chose it because it will eventually be enabled by default. All the more reason for an explanatory paragraph though (or a dedicated section on hooks / the entire package activation process).